tests/integration/components/ds-inline-edit-test.js

```diff
@@ -106,6 +106,58 @@ test('"displayValue" displayed correctly for object model', function(assert){
   }
 })
 
+test('call "customUpdate" when defined', function(assert){
+  const done = assert.async()
+  const { id } = this.server.create('dummy')
+
+  Ember.run(() => this.store.findRecord('dummy', id).then(model => renderComponent.call(this, model)))
+
+  function renderComponent(model){
+    let customUpdateArgs = []
+
+    this.setProperties({
+      model,
+      customUpdate: function(){ customUpdateArgs = arguments }
+    })
+
+    this.render(hbs`
+      {{ds-inline-edit
+        model=model
+        prop="description"
+        id="component"
+        customUpdate=(action customUpdate)
```

Comment:
The "default" action is `confirmEdit`. I want to make it explicit that `customUpdate` customizes the default update